pyproject.toml

```diff
@@ -31,6 +31,8 @@ pyyaml = "*"
 tabulate = "^0.9.0"
 types-tabulate = "^0.9.0.20240106"
 nptdms = "^1.9.0"
+pyjwt = "^2.9.0"
+cryptography = "^43.0.1"
```

Comment:
this library can kinda be a PITA to have as a dependency for a number of reasons from my experience, mostly relating to different OpenSSL versions on various OSs, would prefer for it to not be a primary dep for the core library if we can avoid it